sentry-graphql/build.gradle.kts

```diff
@@ -16,7 +16,7 @@ configure<JavaPluginExtension> {
 
 tasks.withType<KotlinCompile>().configureEach {
     kotlinOptions.jvmTarget = JavaVersion.VERSION_1_8.toString()
-    kotlinOptions.languageVersion = Config.springKotlinCompatibleLanguageVersion
```

Comment:
@maciejwalkowiak cant use 1.3 due to the functional interfaces

Reply:
Spring now supports Kotlin 1.5

Reply:
but then I guess we'd need to bump Spring versions I guess?

Reply:
since this is a separate issue, I will merge, but we could bump the `springKotlinCompatibleLanguageVersion` libs to 1.4 if its already supported (under the v6 branch)